### PR TITLE
Feature/improve namespace

### PR DESCRIPTION
**Pull request by 4lm**
Hi all, 

please review this PR it changes the namespace from "metadata" to "oemetadata". Concerning issue #15 

Beware, 2 of 16 tests are failing, because I didn't change the repo name "yet". Will do if this PR gets accepted ...

Thanks!

### CODE REVIEW

File: tests/metadata/metaschema/draft07/test_schema.py [hunk 1]
```diff
@@ -1,15 +0,0 @@
-def test_if_schema_json_loads_successfully():
```
Comment (Bachibouzouk) [inline tests/metadata/metaschema/draft07/test_schema.py]: Why did you remove these files in a commit with the title "Rename metadata to oemetadata" and not in a subsequent commit?

Comment (Bachibouzouk) [reply tests/metadata/metaschema/draft07/test_schema.py]: ok I understood, renaming the file ...

**Comment by jh-RLI** [comment]
@Ludee I think we should rename the repo to show a clear naming convention for package and repo

**Comment by jh-RLI** [comment]
@Ludee Unfortunately a pytest still causes me problems. For some reason the assertion for the strings "$id" and "$schema" from the schema.json fails. Actually only two strings are compared here. I have tested everything so far and could get a successful run but a second run failed again. I am not sure where the error comes from.

**Comment by jh-RLI** [comment]
Got it there was already "oemetadata" used as repo name in the URL string :)

**Comment by jh-RLI** [comment]
@Ludee Now all tests have been passed and we are ready to merge. But you're gonna have to rename the repo to "oemetadata" because I can't do that.

**Pull request merged**